webpack.config.js

```diff
@@ -133,7 +133,7 @@ module.exports = (env /*: string */ = 'development', options /*: Options */) =>
       new RelayCompilerWebpackPlugin({
         schema: path.resolve(cwd, graphqlConfig.schemaPath),
         src: path.resolve(cwd, opts.srcRoot),
-        watchman: env !== 'production',
```

Comment:
@koddsson `watchman: true` crashes unless `watchman` is in your `PATH`. Disabling for now.